core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java

```diff
@@ -267,13 +261,13 @@ public SshPollValue call() throws Exception {
     
     private SshPollValue exec(String command, Map<String,String> env) throws IOException {
         SshMachineLocation machine = getMachine();
-        Boolean execAsCommand = getConfig(EXEC_AS_COMMAND);
+        Boolean execAsCommand = config().get(EXEC_AS_COMMAND);
         if (log.isTraceEnabled()) log.trace("Ssh polling for {}, executing {} with env {}", new Object[] {machine, command, env});
         ByteArrayOutputStream stdout = new ByteArrayOutputStream();
         ByteArrayOutputStream stderr = new ByteArrayOutputStream();
 
         int exitStatus;
-        ConfigBag flags = ConfigBag.newInstance()
+        ConfigBag flags = ConfigBag.newInstanceExtending(config().getBag())
```

Comment:
@grkvlt this change sneaked in without comment ;-)
It looks reasonable (though the config on the feed will normally not be low-level ssh config, I'd guess). But could you comment on it to say why, and to confirm it is deliberate?